_pages/unit-8-notes.md

```diff
@@ -2,37 +2,39 @@
 
-The method of action research presented by Gill and Johnson (2002) draws on Lewin’s theory of studying things through changing them (i.e., change as the method or process of research).  Stringer and Genat (2004) outline a research process that has action as final step, after eliciting stakeholder perspectives, analyzing data (identifying key features of stakeholder experience), and writing the research report (i.e., change as the outcome of research).  Stringer and Genat also emphasize the participatory process of action research in which change is brought about through the participants in the research project, rather than being imposed on them from the outside.  They explain that a major purpose of participatory approaches to inquiry is to bring people together in a dialogic and productive relationship, enabling the development of a sense of community through the sharing of perspectives, the negotiation of meaning, and the development of collaboratively produced activities, programs, and projects (Stringer & Genat, 2004, p. 9).
+Glanz \(1998\) has suggested that an action research cycle consists of six steps, which may or may not be always addressed in order.  The usual first step is to select a focus.  This is typically followed by data collection, then the analysis and interpretation of the data.  The analysis and interpretation of data typically would suggest action to be taken.  Action is followed by a period of reflection on the results of the action, and on the basis of this reflection, the action is either continued or modified. The cycle then comes full circle to begin with a new focus \(Glanz, 1998, p. 27\).
 
-Glanz (1998) has suggested that an action research cycle consists of six steps, which may or may not be always addressed in order.  The usual first step is to select a focus.  This is typically followed by data collection, then the analysis and interpretation of the data.  The analysis and interpretation of data typically would suggest action to be taken.  Action is followed by a period of reflection on the results of the action, and on the basis of this reflection, the action is either continued or modified.  The cycle then comes full circle to begin with a new focus (Glanz, 1998, p. 27).
+##### References
 
-##### References:
+Gall, M., Gall, J., & Borg, W. \(2007\). _Educational research._ Boston, MA: Pearson.
 
-Gall, M., Gall, J. & Borg, W.  (2007). Educational research.  Boston, MA:  Pearson.
+Gay, L. \(1987\). _Educational research._ Columbus, OH: Merrill.
```

Comment:
This reference should be _Educational Research: Competencies for analysis and application._